app.py

```diff
@@ -3,37 +3,38 @@
 
 from config import config
 
-# base class shared by all models. Needed to instantiate SQLAlchemy object.
-# TODO: find a way to move this to models.py without breaking the app
+
 class BaseModel(Model):
-'''
-Base class to implement common attributes and methods for all models.
-'''
+    """Base class shared by all models to implement common attributes and methods.
+    Needed to instantiate SQLAlchemy object.
+    TODO: find a way to move this to models.py without breaking the app
+    """
+
     @property
     def json(self):
-        '''
-        return json representation of model
-        '''
+        """return json representation of model"""
         return {c.name: getattr(self, c.name) for c in self.__table__.columns}
 
-    #TODO: make this a property instead of a getter method
     @classmethod
     def get_columns(cls):
+        # TODO: make this a property instead of a getter method
         return [c.name for c in cls().__table__.columns]
 
+
 # globally accessible database connection
 db = SQLAlchemy(model_class=BaseModel)
 
 # for some reason doing this import makes models accessible to create_app
 # TODO: see if app can be instantiated without this import
 from models import User, Role, Project, ProjectType, Location
```

Comment:
The master branch doesn't have this... I'm not sure it is needed

Reply:
@t-ketchem do you mean lines 27-29? I agree that they are not necessary. I don't remember why I originally included those imports.

```
# for some reason doing this import makes models accessible to create_app
# TODO: see if app can be instantiated without this import
from models import User, Role, Project, ProjectType, Location
```